src/main.rs

```diff
@@ -1,12 +1,65 @@
-pub mod systems;
+mod constants;
+mod systems;
 
-use crate::systems::{collision::collision_system, velocity::velocity_system};
+use crate::constants::*;
+use crate::systems::{collision::collision_system, round::round_system, velocity::velocity_system};
 use bevy::{prelude::*, render::pass::ClearColor};
 
+pub enum PrevWinner {
+    None,
+    Player(PlayerType),
+}
+pub struct Game {
+    left_score: usize,
+    right_score: usize,
+    prev_winner: PrevWinner,
```

Comment:
`Option<PlayerType>`?

Reply:
Oops forgot to comment, yes obviously 🤦‍♂️ great suggestion. This is one of those rust-idioms that I need to reach for more.